src/chat/chatter.py

```diff
@@ -184,11 +184,25 @@ def update(self):
         url = client.instance.urls.get(player.login)
         if url:
             if url.scheme() == "fafgame":
-                self.statusItem.setIcon(util.icon("chat/status/lobby.png"))
-                self.statusItem.setToolTip("In Game Lobby<br/>"+url.toString())
+                game = client.instance.games.games[int(url.queryItemValue("uid"))]
+                if game.host == self.name:
+                    self.statusItem.setIcon(util.icon("chat/status/host.png"))
+                    self.statusItem.setToolTip("Hosting Game Lobby: &nbsp;" + game.title + "&nbsp; on &nbsp;"
+                                               + game.mapdisplayname + "<br/>" + url.toString())
+                else:
+                    self.statusItem.setIcon(util.icon("chat/status/lobby.png"))
+                    self.statusItem.setToolTip("In Game Lobby: &nbsp;" + game.title + "&nbsp; on &nbsp;"
+                                               + game.mapdisplayname + "<br/>" + url.toString())
             elif url.scheme() == "faflive":
-                self.statusItem.setIcon(util.icon("chat/status/playing.png"))
-                self.statusItem.setToolTip("Playing Game<br/>"+url.toString())
+                game = client.instance.games.games[int(url.path()[0:7])]  # get game_uid from url to get game
```

Comment:
yes it will break, in a view years.
I can handle that right now, would be just a bit more code.